mithila/contact_us.html

```diff
@@ -0,0 +1,127 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact Us</title>
+</head>
+<body>
+
+
+        <header >
+                
+                <table style="background-color:#adc4d4; width: 100%; ">
+                    <tbody>
+                        <tr>
+                            <td style="width:50%;" >
+                                    <a href="index.html" target="_blank"><img src="images/resume-icon.png" width="100px" height="70px"></a>
+                                    
+
+                            </td>
+                            <td style="width:50%;">
+                                   
+                                        
+                                        
+                                       Email: <input type="email"  placeholder="Enter e-mail" autofocus="autofocus" autocomplete="autocomplete">
+                                        Password :<input type="password" placeholder="Enter password">
+                                        <button type="submit">Login</button><b>or</b>
+                                       <a href="sign_in.html" target="_blank"><button type="button">Sign in</button></a> <br>
+                                        
+                                            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
+
+                                              Remember me: <input type="checkbox"> 
+                                       
+
+                            </td>
+
+
+
+
+                        </tr>
+
+
+                    </tbody>
+                                    
+                
+                 </table>
+
+            </header>  
+            
+           
+            <div style="padding-left: 200px;margin-top: 93px;">
+
+                                                               
+                            <h2>Contact us via email </h2>
+                            <p><i>Feel free for any quarry</i></p>
+
+                        
+                        <form>
+                            <label><b>Full name:</b></label>
+                            <input type="text" name="full_name" placeholder="Enter full name" required="required">
+
+                            <br><br>
+
+
+                        
+                            <label><b>Email:</b></label>
+                            <input type="email" placeholder="Enter your email" name="email"  required="required">
+
+                            <br><br>
+
+                            <label><b>Subject</b></label>
+                            <input type="text" name="full_name" placeholder="Enter email Subject" required="required">
+
+                            <br><br>
+
+
+                            <label><b>Select Degree:</b></label>
+                                <select name="degree">
+
+                                    
+                                    <option selected="selected">Select one</option>
```

Comment:
don't need`selected="selected"` cause first item is default selected.. You can select the second item